xml/System.Buffers.Binary/BinaryPrimitives.xml

```diff
@@ -2587,8 +2597,8 @@ Reads exactly 4 bytes from the beginning of the span.
       <Docs>
         <param name="source">To be added.</param>
         <param name="value">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Reads a <see cref="T:System.UInt128" /> from the beginning of a read-only span of bytes, as little endian.</summary>
+        <returns>If the span is too small to contain a <see cref="T:System.UInt128" />, return <see langword="false" />.</returns>
```

Comment:
```suggestion
        <returns><see langword="false" /> if the span is too small to contain a <see cref="T:System.UInt128" />; otherwise, <see langword="true" />.</returns>
```